Source/SocketEngine.swift

```diff
@@ -151,8 +151,7 @@ public final class SocketEngine : NSObject, SocketEnginePollable, SocketEngineWe
         if message.hasPrefix("b4") {
             // binary in base64 string
             let noPrefix = message[message.characters.index(message.startIndex, offsetBy: 2)..<message.endIndex]
-
-            if let data = Data(base64Encoded: noPrefix, options: Data.Base64EncodingOptions(rawValue: 0)) {
+            if let data = Data(base64Encoded: noPrefix, options: Data.Base64DecodingOptions(rawValue: 0)) {
```

Comment:
This was the only actual error. Does this seem correct? `Data.Base64EncodingOptions` => `Data.Base64DecodingOptions`